flowz/channels/core.py

```diff
@@ -10,14 +10,39 @@
 
 from flowz import util
 
+
 class ChannelDone(Exception):
     """
     Exception throw when trying to access a completed channel.
 
     This is akin to StopIteration, but channel-oriented.  We can't
     use StopIteration because `tornado.gen.coroutine` swallows those.
     """
-    pass
+
+    def __init__(self, *args, **kwargs):
```

Comment:
I'm not following why this is necessary; did you have this in there for debugging and forget to remove it?